index.html

```diff
@@ -578,10 +578,10 @@ <h3>Terminology</h3>
           <dt><dfn class="export">verification method</dfn></dt>
           <dd>
             <p>
-A set of parameters that can be used together with a process to independently
-verify a proof. For example, a cryptographic public key can be used as a
-verification method with respect to a digital signature; in such use, it
-verifies that the signer possessed the associated cryptographic private key.
+A method and its parameters used to independently verify a proof. For
+example, a cryptographic public key can be used as a verification method with
+respect to a digital signature; in such use, it verifies that the signer
+possessed the associated cryptographic private key.
```

Comment:
I'd like to keep pushing the community to understand that possession of the key isn't required, only access to an interface that will make use of it. This is a perhaps subtle but important shift in thinking and helps better inform security/threat modeling.

```suggestion
used the associated cryptographic private key.
```